boards/arduino-atmega-common/include/arduino_pinmap.h

```diff
@@ -156,6 +156,26 @@ extern "C" {
 #define ARDUINO_PIN_A15         ARDUINO_PIN_69
 #endif
 
+/* arduino ADC notation compatibility */
+#define A0                     0
```

Comment:
Don't define macros with short names like this, it has the potential to create some very hard to track down problems later on when users try to use A0 etc as a variable name.